Rakefile

```diff
@@ -11,6 +11,8 @@ namespace :resque do
   end
 
   task clean_working_job_ids: :prune_dead_workers do
+    next if ENV['APP_ENV'] == 'production'
```

Comment:
Don't we want to prune dead workers even in production ?

Reply:
Containers cannot see legit workers of other cont.

Reply:
This task is not really needed on the server-side anymore (thanks to the fact that docker workers continue until they finish the job at hand and then set the job status to completed...) it is still of use on local to prevent accumulation of jobs (ergo the `if ENV['APP_ENV'] == 'production'` condition)